src/pages/ProductPage.jsx

```diff
@@ -0,0 +1,13 @@
+import Product from "../api/Product";
+import BreadCrumbs from "../components/breadcrumbs/BreadCrumbs";
+
+const ProductPage = () => {
+  return (
+    <div className="product-page">
```

Comment:
Generally speaking its better to fetch the data in the page and pass it to components as a prop. This makes testing a lot easier in future.

Reply:
Ok. I will do that. Thanks for the information.